packages/template-typescript-minimal/package.json

```diff
@@ -45,7 +49,185 @@
   },
   "mobify": {
     "app":{
-      "extensions": ["@salesforce/extension-sample"]
+      "extensions": [
+        "@salesforce/extension-sample",
+        "@salesforce/extension-store-locator",
+
+        ["@salesforce/extension-chakra-storefront", {
+          "enabled": true,
+          "routes": [
+            {
+              "page": "Home",
+              "path": "/",
+              "exact": true
+            },
```

Comment:
For the chakra-storefront extension, the base project needs to provide a valid working config though. So that's why this is here.